corehq/apps/domain/templates/domain/renew_plan.html

```diff
@@ -1,5 +1,6 @@
 {% extends "domain/base_change_plan.html" %}
 {% load hq_shared_tags %}
+
```

Comment:
It did not, in fact, get linted. Need to work on the github action/workflow it seems.

Reply:
On the plus side, it doesn't seem to have broken existing JS linting via the github action.